gen3/bin/kube-setup-secrets.sh

```diff
@@ -208,7 +208,25 @@ if [[ -f "${WORKSPACE}/${vpc_name}_output/creds.json" ]]; then  # update secrets
     g3kubectl create secret generic sheepdog-secret "--from-file=wsgi.py=${GEN3_HOME}/apis_configs/sheepdog_settings.py" "--from-file=${GEN3_HOME}/apis_configs/config_helper.py"
   fi
 
-  #
+  if ! g3kubectl get secret aws-creds-secret > /dev/null 2>&1; then
+    g3kubectl create secret generic aws-creds-secret --from-file=credentials=./apis_configs/aws_creds_secret
+  fi
+
+  if ! g3kubectl get secrets/dcf-dataservice-json-secret > /dev/null 2>&1; then
+    if [[ ! -f "./apis_configs/dcf_dataservice_credentials.json" ]]; then
+        touch "./apis_configs/dcf_dataservice_credentials.json"
+    fi
+    echo "create dcf-dataservice-json-secret using current creds file apis_configs/dcf_dataservice_credentials.json"
+    g3kubectl create secret generic dcf-dataservice-json-secret --from-file=dcf_dataservice_credentials.json=./apis_configs/dcf_dataservice_credentials.json
+  fi
+
+  if ! g3kubectl get configmaps/data-service-manifest > /dev/null 2>&1; then
+    if [[ ! -f "./apis_configs/manifest" ]]; then
+      touch "./apis_configs/manifest"
+    fi
+    g3kubectl create configmap data-service-manifest --from-file=manifest=./apis_configs/manifest
+  fi
+
   # Create the 'sheepdog' and 'peregrine' postgres user if necessary
```

Comment:
We don't need these secrets and configurations in most commons, so we should only create them if necessary.  Move all these confile file to be under one directory - `apis_configs/dcf_dataservice/creds.json` and `apis_configs/dcf_dataservice/manifest`, and wrap this whole block in an `if` block:
```
if [[ -d apis_configs/dcf_dataservice ]]; then ...
```

Reply:
fixed at e9fdab74ac2aa33db7836eb27afa2890695ce597